Source/Core/Core/Core.cpp

```diff
@@ -535,7 +535,15 @@ static void EmuThread(std::unique_ptr<BootParameters> boot, WindowSystemInfo wsi
 
   Common::ScopeGuard sd_folder_sync_guard{[sync_sd_folder] {
     if (sync_sd_folder && Config::Get(Config::MAIN_ALLOW_SD_WRITES))
-      Common::SyncSDImageToSDFolder([]() { return false; });
+    {
+      bool sync_ok = Common::SyncSDImageToSDFolder([]() { return false; });
+      if (!sync_ok)
+      {
+        PanicAlertFmt("Failed to sync SD card with folder. All changes made this session will be "
+                      "discarded on next boot if you do not manually re-issue a resync in Config > "
+                      "Wii > SD Card Settings > Convert File to Folder Now!");
```

Comment:
We want the message to be translatable:

```suggestion
        PanicAlertFmtT("Failed to sync SD card with folder. All changes made this session will be "
                       "discarded on next boot if you do not manually re-issue a resync in Config > "
                       "Wii > SD Card Settings > Convert File to Folder Now!");
```

Reply:
Good point--fixed.